OpenEdXMobile/build.gradle

```diff
@@ -149,6 +149,7 @@ dependencies {
     implementation 'com.google.android.gms:play-services-cast-framework:16.2.0'
 
     implementation 'com.facebook.android:facebook-login:5.9.0'
+    implementation 'com.microsoft.identity.client:msal:0.3+'
```

Comment:
Why are we using an old version? and also a `+` instead of a fully resolved version?

p.s. the latest version according to [docs](https://docs.microsoft.com/en-us/azure/active-directory/develop/tutorial-v2-android#use-msal) is: `implementation 'com.microsoft.identity.client:msal:1.+'`

Reply:
We need to change the overall implementation to upgrade the `client:msal` version.

Reply:
Ok, I would suggest creating a ticket for the up gradation then.

Reply:
Ok, I would suggest creating a ticket for the up gradation then.

Reply:
Ticket has been created [LEARNER-7712](https://openedx.atlassian.net/browse/LEARNER-7712)